### ISSUE TITLE
Support for DemoStorage + MappingStorage

### ISSUE THREAD

**Issue by azmeuk** [Pylons/zodburi#23]
It is very useful to use a `DemoStorage` on top of a `MappingStorage`, for instance in unit testing with support for conflicts resolution (`MappingStorage` cannot solve conflicts, but `DemoStorage+MappingStorage` can).
Thus, I would love zodburi to handle this.
```python
import zodburi
factory, args = zodburi.resolve_uri("memory://storagename?demostorage=true")
# KeyError: 'Unrecognized database keyword(s): demostorage'
```

What do you think?

**Comment by navytux** [comment]
See also https://github.com/Pylons/zodburi/pull/29.

**Comment by bertjwregeer** [comment]
This has been merged with #29, @navytux I'll leave it in your hands to do a release. Let me know if you need any help!

**Comment by navytux** [comment]
Thanks, @bertjwregeer. I've followed-up to make the next release in https://github.com/Pylons/zodburi/pull/29#issuecomment-836661524.
